### ISSUE TITLE
how to ignore the root of the system

### ISSUE THREAD

**Issue by Manix80** [hrsh7th/cmp-path#68]
hi,

is it possible not to display the root of the system? for example: /sys ...

when I type / I automatically get the root.

in the doc there is the option "get_cwd" but I don't know how to do it.

the best would be to be able to display the current directory.

I have the same configuration as here: https://github.com/sar/cmp.nvim#setup

with the exception that I added this:

```
cmp.setup({
...
  sources = cmp.config.sources({
    ...
    { name = "path" },
    ...
  )}
...
})
```

thanks

**Comment by liquiddandruff** [comment]
Yeah, at the moment using default configs this is causing `:e <anything>` to hang for 1 minute on WSL2, it's finding things like random DLLs from IDK where. Need to make this configurable otherwise it's not usable atm.

From looking at the source it doesn't appear there's a way to do this atm.

**Comment by Manix80** [comment]
I haven't found a solution for this problem, which isn't a problem in fact, just a habit to get into and be a little more careful with a "/".

**Comment by NullVoxPopuli** [comment]
I just ran in to this and had to disable `cmp-path` -- I thought neovim was freezing whenever I typed a comment (`//` or `/*`) -- or a self-closing element (`<br />`)
